deployment/ib-kubernetes-ds.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ib-kubernetes
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ib-kubernetes
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "patch", "watch"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["*"]
+    verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ib-kubernetes
+roleRef:
+  name: ib-kubernetes
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: ib-kubernetes
+    namespace: kube-system
+---
+# Runs daemon on the master node(s)
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ib-kubernetes
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches the ib-kubernetes daemon for infiniband cni.
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ib-kubernetes
+  template:
+    metadata:
+      labels:
+        name: ib-kubernetes
+    spec:
+      serviceAccountName: ib-kubernetes
+      hostNetwork: true
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+        kubernetes.io/os: "linux"
+      containers:
+        - name: ib-kubernetes
+          image: mellanox/ib-kubernetes
+          imagePullPolicy: IfNotPresent
+          command: ["/usr/bin/ib-kubernetes"]
+          args:
+            - "-logtostderr"
+            - "-v=3"
```

Comment:
you set verbose level to 3 but dont use the V style logging in the code.
does that disable the log prints you currently have in code? 

e.g to my understanding you should: glog.V(2).Infof("some log")
then setting -v=2 would make  them visible and -v=3 would hide them